src/services/itemMembership/membership.controller.ts

```diff
@@ -14,31 +13,24 @@ import { ItemMembershipService } from './membership.service';
 import MembershipRequestAPI from './plugins/MembershipRequest/membershipRequest.controller';
 import { membershipWsHooks } from './ws/hooks';
 
-const ROUTES_PREFIX = '/item-memberships';
-
 const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
   const itemMembershipService = resolveDependency(ItemMembershipService);
 
   fastify.register(MembershipRequestAPI, {
-    prefix: '/items/:itemId/memberships/requests',
+    prefix: '/:itemId/memberships/requests',
   });
```

Comment:
Should this be registered inside the membershiups plugin so it benefits from the common path prefix ?